app/src/main/java/com/example/henzoshimada/feeltrip/MainScreen.java

```diff
@@ -197,6 +239,25 @@ private void addItemsOnEmotionalStateSpinner(){
                 android.R.layout.simple_spinner_item, emotionalStateList);
         emotionalStateAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
         emotionalStateSpinner.setAdapter(emotionalStateAdapter);
+        emotionalStateSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener()
+        {
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id)
+            {
+                String selectedItem = parent.getItemAtPosition(position).toString();
```

Comment:
Could the spinner call a horizontalscrollview? Then we can just pass in my dynamic scrollbar to here as well. Either way, my vision was to have the horizontalscrollbar open up upon clicking the drop down arrow for filter by moods, and then having some green textview display the current selected option.